pkg/rewards/claim.go

```diff
@@ -5,6 +5,8 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/Layr-Labs/eigenlayer-rewards-proofs/pkg/distribution"
```

Comment:
can you move imports to right place? if you see we keep imports clustered properly so it's easy to see it. Eventually we want to have this fail in CI, we should use a tool like [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports)

Reply:
Ah, thanks, I'll change this.